SlottyMedia.DatabaseSeeding/ImageDownloader.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using SlottyMedia.DatabaseSeeding;
+using SlottyMedia.LoggingProvider;
+
```

Comment:
namespace SlottyMedia.DatabaseSeeding;

Reply:
Otherwise xmldoc2md won't find all references in the assembly hence it will throw a error on a object null reference

Reply:
This should now be resolved